actions/create_archive_dir.yaml

```diff
@@ -0,0 +1,43 @@
+---
+    name: 'create_archive_dir'
+    runner_type: 'run-local'
+    description: 'Creates a copy of the runfolder, consisting of symlinks, for archiving purposes.'
+    enabled: true
+    entry_point: ''
+    parameters:
+        timeout:
+            default: 86400
+        runfolder:
+            type: 'string'
+            description: 'Path to runfolder to archive'
+            required: true
+        fastq_threshold:
```

Comment:
This is just a rudimentary way of ensuring that the `Unaligned` directory actually contains some data (i.e. FASTQ files). So I'm just counting them, and the workflow will abort if less than the number of expected files are found.